.changeset/short-pumpkins-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-ldap': minor
```

Comment:
```suggestion
'@backstage/plugin-catalog-backend-module-ldap': patch
```

Since we're on `0.x` of this package, any non breaking changes fall under `patch`